web/router.ex

```diff
@@ -127,10 +127,11 @@ defmodule HexWeb.Router do
     get    "/packages",       PackageController, :index
     get    "/packages/:name", PackageController, :show
 
-    get    "/packages/:name/releases/:version", ReleaseController, :show
-    delete "/packages/:name/releases/:version", ReleaseController, :delete
+    get    "/packages/:name/releases/:version",        ReleaseController, :show
+    delete "/packages/:name/releases/:version",        ReleaseController, :delete
+    post   "/packages/:name/releases/:version/retire", ReleaseController, :retire
+    delete "/packages/:name/releases/:version/retire", ReleaseController, :unretire
```

Comment:
Wouldn't it be better to have a separate `RetirementController`? Those actions would fit perfectly into the regular create and delete actions.